docs/src/content/docs/commands/JSON.STRLEN.md

```diff
@@ -1,43 +1,49 @@
 ---
 title: JSON.STRLEN
-description: Documentation for the DiceDB command JSON.STRLEN
+description: The 'JSON.STRLEN' command is used to get the length of a string at a given path in a JSON Document stored in DiceDB
 ---
 
 The `JSON.STRLEN` command is part of the DiceDBJSON module, which allows you to work with JSON data in DiceDB. This command is used to determine the length of a JSON string at a specified path within a JSON document stored in DiceDB.
```

Comment:
Hi, DiceDB supports JSON natively and hence do not require DiceDBJSON module for this command to work. Please remove any such mention from this documentation. Thanks